pages/common/2to3.md

```diff
@@ -11,7 +11,7 @@
 
 `2to3 --write {{path/to/file.py}}`
 
-- Convert specific Python 2 language features to Python 3:
+- Convert a specific Python 2 language features to Python 3:
```

Comment:
`a` shouldn't be here (`a language features`)

Reply:
Ahh yeah, thanks for that.
No idea how I did that.

```suggestion
- Convert specific Python 2 language features to Python 3:
```